src/ripple/protocol/impl/Issue.cpp

```diff
@@ -43,60 +43,33 @@ operator<<(std::ostream& os, Issue const& x)
     return os;
 }
 
-/** Ordered comparison.
-    The assets are ordered first by currency and then by account,
-    if the currency is not XRP.
-*/
-int
-compare(Issue const& lhs, Issue const& rhs)
+std::weak_ordering
```

Comment:
Note that `std::weak_ordering` is the correct thing here, unlike the previous case of `base_uint`.